internal/hmyapi/backend.go

```diff
@@ -1,22 +1,21 @@
 package hmyapi
 
 import (
-	"github.com/harmony-one/harmony/rpc"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/harmony-one/harmony/core"
 )
 
-const namespace = "hmy"
-
 // GetAPIs returns all the APIs.
-func GetAPIs(b *rpc.HmyAPIBackend) []rpc.API {
+func GetAPIs(b *core.HmyAPIBackend) []rpc.API {
 	nonceLock := new(AddrLocker)
 	return []rpc.API{
 		{
-			Namespace: namespace,
+			Namespace: "hmy",
```

Comment:
why not use const?

Reply:
may add some other API with different namespace. so didn't use const.

Reply:
But for this hmy namespace I still think we should make it a const.